src/style.css

```diff
@@ -6,7 +6,8 @@
 * {
   box-sizing: border-box;
   padding: 0;
-  margin: 0;
+  max-width: 500px;
+  margin: auto;
 }
 
 body,
```

Comment:
Wireframe
![tempsnip](https://github.com/Bleu-Yves-Sopoude/Leaderboard/assets/78251927/8982310c-b3dd-4080-8b5e-e08679cfc047)

Your design
![image](https://github.com/Bleu-Yves-Sopoude/Leaderboard/assets/78251927/e2f7c06c-2abb-4d00-9b42-ef03c952f2fa)

Please keep the general layout of the wireframe, according to the [project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m3_final_touches.md) 

Kindly modify your styling to make your design look like the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png)🤓

- The main title `Leaderboard` is aligned to the left

- The page is divided `equally` into two parts which are
  - A section to display users' scores
  - A section for `The form`

![image](https://github.com/Bleu-Yves-Sopoude/Leaderboard/assets/78251927/41b88b7f-4b5b-40db-a282-507822c1a9d5)

- [OPTIONAL] - I recommend you change the `text` color for a better contrast ratio between the text color and its background to ensure that the text is easy to read for all users, including those with visual impairments, improving the user's experience of your website. 💪 👍 

## Form
Wireframe
![image](https://github.com/Bleu-Yves-Sopoude/Leaderboard/assets/78251927/968e0365-8246-42f4-9842-a8a8f6b7e2ee)

Your design
![image](https://github.com/Bleu-Yves-Sopoude/Leaderboard/assets/78251927/5e7bccf3-d605-4323-bbb4-02d6479c0d82)
- Please adjust the alignment so that all the form inputs are aligned to the `right` according to the wireframe
- It will be nice if you increase the width of the submit button so that the text content `Submit score` stay on a single line 🤗👍